src/host_toolchains.py

```diff
@@ -26,8 +26,8 @@
 SETUP_TOOLCHAIN = os.path.join(CR_BUILD_DIR, 'toolchain', 'win',
                                'setup_toolchain.py')
 V8_SRC_DIR = os.path.join(WORK_DIR, 'v8', 'v8')
```

Comment:
Done.

When V8 migrated from GYP to GN last year, `v8\gypfiles` and some `.gyp` files was not removed because some external embedders like Node.js still depend on GYP. However, V8 team decided that maintaining both GYP and GN build together is too much burden, so end of this year all GYP files will be removed from V8. The burden to maintain GYP is shifted to Node.js.